packages/frames-client/src/converters.ts

```diff
@@ -0,0 +1,43 @@
+import { publicKey } from "@xmtp/proto";
```

Comment:
I'm trying to avoid actually importing `xmtp-js`, so I just re-implemented some functionality for converting V1 bundles to V2 bundles here.